src/entry.ts

```diff
@@ -55,6 +83,69 @@ function isSkippableField(field: Field): boolean {
   );
 }
 
+function appendCheckbox(entryList: EntryList, field: Field): void {
```

Comment:
I would put `appendCheckbox` below `appendEntry` just so the documentation will explain the `appendEntry` first.

Reply:
I was organizing the code in a depth-first manner to avoid arbitrary placement of components. I can add some docs here, if you think it will help.

Reply:
Had to dig two levels to get the documentation I would have liked to see there so wouldn't hurt to have docs on all the append functions.

Reply:
How's that? Even added a link for you 😃

Reply:
It's good now 😄